src/app/public/params-provider.ts

```diff
@@ -25,11 +30,12 @@ export class SkyAppRuntimeConfigParamsProvider {
   private _params: SkyAppRuntimeConfigParams;
 
   constructor(
-    config: SkyAppParamsConfig
+    @Optional() paramsConfig?: SkyAppParamsConfig,
+    @Optional() configParams?: SkyAppConfigParams
   ) {
     this._params = new SkyAppRuntimeConfigParams(
       window.location.href,
-      config.params
+      paramsConfig?.params || configParams?.params || {}
```

Comment:
<img width="506" alt="Screen Shot 2021-03-08 at 10 31 55 AM" src="https://user-images.githubusercontent.com/12497062/110342562-88954400-7ff9-11eb-8285-6ec07a673394.png">
☝️ How it looks in the IDE.